wallets/react-wallet-v2/README.md

```diff
@@ -48,3 +48,54 @@ Your `.env.local` now contains the following environment variables:
 ## Preview of wallet and dapp examples in action
 
 https://user-images.githubusercontent.com/3154053/156764521-3492c232-7a93-47ba-88bd-2cee3f8366d4.mp4
+
+## DOM Tags
+This section is a key of the dom elements used for internal E2E testing
+Accessible with `data-testid`
+
+### Navbar
+| Key | Description |
+| ----------- | ----------- |
+|  `accounts` | Accounts page |
+| `sessions` | Sessions page |
+| `wc-logo` |  WC Connect page |
```

Comment:
Minor: I’d keep this aligned with the underlying functionality like with the other page IDs. `wc-logo` doesn’t convey what that page does.

```suggestion
| `wc-connect` |  WC Connect page |
```